packages/rest/src/rest.server.ts

```diff
@@ -244,16 +244,9 @@ export class RestServer extends Context implements Server, HttpServerLike {
       );
     }
 
-    const explorerConfig = this.config.apiExplorer || {};
-    if (explorerConfig.disabled) {
-      debug('Redirect to swagger-ui was disabled by configuration.');
-      return;
-    }
-
     const explorerPaths = ['/swagger-ui', '/explorer'];
```

Comment:
Should `explorerPaths` take into account custom paths the user configures with our `RestExplorer` component?

Reply:
I am not opposed to that, but I think it's out of scope of this pull request.